scipy/special/add_newdocs.py

```diff
@@ -3036,28 +3036,35 @@ def add_newdoc(place, name, doc):
 
     Gauss hypergeometric function 2F1(a, b; c; z).
 
+    \mathrm{hyp2f1}(a, b, c, z) = 1 + \sum_{k=0}^inf. \frac {{{a+k}\choose{a}} {{b+k}}\choose{b}}}{{c+k}\choose{c}} z^(k+1).
+
     Parameters
     ----------
-    a : double argument.
+    a : array_like
+        Parameter
 
-    b : double argument.
+    b : array_like
+        Parameter
 
-    c : double argument.
+    c : array_like
+        Parameter
 
-    z : double argument.
+    z : array_like
+        Parameter
 
     Returns
     -------
-    y : double argument
-        y being the solution to the second-order linear ordinary differential equation.
+    y : ndarray
+        y being the values of the gaussian hypergeometric function.
 
     Notes
     -----
 
-    .. math::
+    Gaussian function is defined on the disk |z|<1, and by analytic continuation elsewhere.
```

Comment:
The Gaussian function is `e^(-x**2)`. Should be Gaussian hypergeometric function.